Proteomics/ModificationWithLocation.cs

```diff
@@ -64,20 +64,11 @@ public override bool Equals(object o)
             return m == null ? false :
 
                base.Equals(m)
-               && (this.accession == null && m.accession == null
```

Comment:
Actually, following that logic, I should remove the comparison of id's as well...